web/translation/translate.es_ES.toml

```diff
@@ -147,8 +147,8 @@
 "totalFlow" = "Flujo Total"
 "leaveBlankToNeverExpire" = "Dejar en Blanco para Nunca Expirar"
 "noRecommendKeepDefault" = "No hay requisitos especiales para mantener la configuración predeterminada"
-"certificatePath" = "Camino"
-"certificateContent" = "Contenido"
+"certificatePath" = "Ruta del Archivo"
+"certificateContent" = "Contenido del Archivo"

```

Comment:
These two are too long , please make it shorter

Reply:
already fixed. :D